### ISSUE TITLE
Support for Hetzner Cloud Controller Manager on Dedicated Bare Metal Servers

### ISSUE THREAD

**Issue by kolovo** [hetznercloud/hcloud-cloud-controller-manager#712]
### TL;DR

Does the Hetzner Cloud Controller Manager (CCM) support Kubernetes clusters on dedicated bare metal servers, or is it limited to VMs on Hetzner Cloud?

Hello,

I am using a Kubernetes cluster running on dedicated bare metal servers from Hetzner. I would like to know if the Hetzner Cloud Controller Manager (CCM) supports integration with Kubernetes clusters on dedicated servers, or if it is exclusively designed for use with VMs on Hetzner Cloud.

### Expected behavior

- The CCM should be able to manage LoadBalancer services on Kubernetes clusters running on dedicated bare metal servers.
- Clear documentation or guidance on any additional configuration needed to enable this support for bare metal setups.

Any guidance or documentation on this would be greatly appreciated.

Thank you!

**Comment by apricote** [comment]
Hey @kolovo,

HCCM supports dedicated bare metal servers from Hetzner Robot. You can find the docs [here](https://github.com/hetznercloud/hcloud-cloud-controller-manager/blob/main/docs/robot.md).

**Comment by kolovo** [comment]
Hi @apricote ,
Thanks a lot for the fast response .
Sorry for didnt check further on the repo  .
Ticket can be closed .
